### ISSUE TITLE
[Bug]Crash after tapping (required reading) link from uBlock settings

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#8140]
### Prerequisites
Have uBlock Origin installed.

## Steps to reproduce
1. From the Add-ons manager tap on uBlock Origins
2. Tap and open "Settings"
3. From the Settings section tap on the link from (required reading)

### Expected behavior
The link should be properly opened.

### Actual behavior
Native code crash.

### Device information

* Android device: 
• Google Pixel 3a XL (Android 10)
• Google Pixel 3a (Android 9)
• Huawei Mate 20 Lite (Android 8.1.0)
• Samsung Galaxy S7 (Android 7.0.0)
• OnePlus A3 (Android 6.0.1)
• LG Nexus 4 (Android 5.1.1)

* Fenix version: Nightly #20370604 from 2/6

### Notes

<details>
    <summary>Video comparison between Fennec VS Fenix (dropdown)</summary>

**►Fenix**
![20200206_090948](https://user-images.githubusercontent.com/51314259/73914406-e79dfc00-48c1-11ea-8ffc-321f9c8523ee.gif)

**►Fennec**
![20200206-100613](https://user-images.githubusercontent.com/51314259/73917823-e15f4e00-48c8-11ea-9916-094f88ecd2a8.gif)
**

</details>

► Log
```kotlin
--------- beginning of system
02-06 09:13:03.774 1279-2223/? I/ActivityManager: Start proc 11963:org.mozilla.fenix.nightly:mozilla.components.lib.crash.CrashHandler/u0a210 for service org.mozilla.fenix.nightly/mozilla.components.lib.crash.handler.CrashHandlerService
02-06 09:13:03.775 11955-11955/? D/AndroidRuntime: Shutting down VM
02-06 09:13:03.956 11963-11976/? E/mozac/CrashReporter: CrashHandlerService received native code crash
02-06 09:13:03.956 11963-11976/? I/mozac/CrashReporter: Received crash: NativeCodeCrash(minidumpPath=/data/user/0/org.mozilla.fenix.nightly/files/mozilla/b41h6vuu.default/minidumps/7eebea6e-f2ff-5956-b40c-542f677b3377.dmp, minidumpSuccess=false, extrasPath=/data/user/0/org.mozilla.fenix.nightly/files/mozilla/b41h6vuu.default/minidumps/7eebea6e-f2ff-5956-b40c-542f677b3377.extra, isFatal=true, breadcrumbs=[])
02-06 09:13:03.958 11963-11976/? I/mozac/CrashReporter: Showing prompt
02-06 09:13:03.961 1279-2223/? I/ActivityManager: START u0 {flg=0x10018000 cmp=org.mozilla.fenix.nightly/mozilla.components.lib.crash.prompt.CrashReporterActivity (has extras)} from uid 10210 pid 11963 on display 0
02-06 09:13:04.038 11963-11976/? I/art: System.exit called, status: 0
02-06 09:13:04.038 11963-11976/? I/AndroidRuntime: VM exiting with result code 0, cleanup skipped.
02-06 09:13:04.085 1279-1835/? I/ActivityManager: Process org.mozilla.fenix.nightly:mozilla.components.lib.crash.CrashHandler (pid 11963) has died
02-06 09:13:04.086 1279-1835/? W/ActivityManager: Scheduling restart of crashed service org.mozilla.fenix.nightly/mozilla.components.lib.crash.handler.CrashHandlerService in 1000ms
02-06 09:13:04.091 1279-1328/? V/ActivityManager: killProcessGroupAsync took 6 ms for PID 11963 on thread 14
02-06 09:13:04.106 1279-1292/? I/art: Background partial concurrent mark sweep GC freed 31538(1957KB) AllocSpace objects, 12(268KB) LOS objects, 32% free, 32MB/48MB, paused 1.005ms total 102.961ms
02-06 09:13:04.114 1279-2595/? I/WindowState: WIN DEATH: Window{36b953a u0 org.mozilla.fenix.nightly/org.mozilla.fenix.nightly.App}
02-06 09:13:04.115 1279-2595/? W/WindowManager: Force-removing child win Window{3f5eeee u0 SurfaceView} from container Window{36b953a u0 org.mozilla.fenix.nightly/org.mozilla.fenix.nightly.App}
02-06 09:13:04.115 1279-2800/? I/ActivityManager: Process org.mozilla.fenix.nightly (pid 11677) has died
02-06 09:13:04.115 1279-1314/? D/GraphicsStats: Buffer count: 8
02-06 09:13:04.118 11746-11746/? I/ServiceChildProcess: Service has been unbound. Stopping.
02-06 09:13:04.121 1279-2224/? W/WindowManager: Failed looking up window
    java.lang.IllegalArgumentException: Requested window android.os.BinderProxy@41ea684 does not exist
        at com.android.server.wm.WindowManagerService.windowForClientLocked(WindowManagerService.java:8805)
        at com.android.server.wm.WindowManagerService.windowForClientLocked(WindowManagerService.java:8796)
        at com.android.server.wm.WindowState$DeathRecipient.binderDied(WindowState.java:1228)
        at android.os.BinderProxy.sendDeathNotice(Binder.java:582)
02-06 09:13:04.121 1279-2224/? I/WindowState: WIN DEATH: null
02-06 09:13:04.133 1279-2800/? V/BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@992de25
02-06 09:13:04.133 1279-2800/? I/ActivityManager: Start proc 11978:org.mozilla.fenix.nightly:mozilla.components.lib.crash.CrashReporter/u0a210 for activity org.mozilla.fenix.nightly/mozilla.components.lib.crash.prompt.CrashReporterActivity
02-06 09:13:04.136 11746-11746/? I/Process: Sending signal. PID: 11746 SIG: 9
02-06 09:13:04.157 1279-1818/? I/ActivityManager: Process org.mozilla.fenix.nightly:tab (pid 11746) has died
02-06 09:13:04.167 1279-2803/? I/ActivityManager: Process org.mozilla.fenix.nightly:socket (pid 11765) has died
02-06 09:13:04.276 724-724/? I/Zygote: Process 11677 exited cleanly (11)
02-06 09:13:04.278 1279-1328/? V/ActivityManager: killProcessGroupAsync took 161 ms for PID 11677 on thread 14
02-06 09:13:04.278 1279-1328/? V/ActivityManager: killProcessGroupAsync took 0 ms for PID 11746 on thread 14
02-06 09:13:04.279 1279-1328/? V/ActivityManager: killProcessGroupAsync took 0 ms for PID 11765 on thread 14
02-06 09:13:04.301 1279-2800/? W/InputMethodManagerService: Got RemoteException sending setActive(false) notification to pid 11677 uid 10210
02-06 09:13:04.450 11978-11978/? D/AppTracker: App Event: start
02-06 09:13:04.461 11978-11994/? D/OpenGLRenderer: Use EGL_SWAP_BEHAVIOR_PRESERVED: true
02-06 09:13:04.516 11978-11994/? I/Adreno: QUALCOMM build                   : 19f8f74, Iac38d51750
    Build Date                       : 06/06/16
    OpenGL ES Shader Compiler Version: XE031.07.00.01
    Local Branch                     : case2489916_Unmap_crash
    Remote Branch                    : 
    Remote Branch                    : 
    Reconstruct Branch               : 
02-06 09:13:04.522 11978-11994/? I/OpenGLRenderer: Initialized EGL, version 1.4
02-06 09:13:04.540 11978-11978/? D/AbstractTracker: Event success
02-06 09:13:04.553 11978-11994/? E/HAL: load: gralloc.msm8996.so
02-06 09:13:04.580 1279-1352/? I/ActivityManager: Displayed org.mozilla.fenix.nightly/mozilla.components.lib.crash.prompt.CrashReporterActivity: +459ms
02-06 09:13:05.436 1279-1279/? I/DeviceIdleController: updateChargingLocked: charging=true
02-06 09:13:05.437 1279-1279/? I/DeviceIdleController: becomeActiveLocked, reason = charging

```

**Comment by csadilek** [comment]
GV bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1613237

**Comment by csadilek** [comment]
@AndiAJ This crash is fixed now in the latest Nightly. Can you verify, please?

Note that since we're putting the Settings page in a popup, navigation is limited. Once a link is opened the device back button can be used to go back, but that's about it. The settings page is currently being redesigned and we might also end up putting it in a regular tab for better navigation, but that's a separate discussion :)

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on the latest **`Nightly Build #20550606 from 02/24`** using the following devices:
• Google Pixel 3a XL (Android 10)
• Google Pixel 3a (Android 9)
• Huawei Mate 20 Lite (Android 8.1.0)
• Samsung Galaxy S7 (Android 7)
• OnePlus A3 (Android 6.0.1)
• LG Nexus 4 (Android 5.1.1)

► Video
![20200224-114259](https://user-images.githubusercontent.com/51314259/75142385-a304c400-56fb-11ea-9e83-527a02ae5478.gif)
